migrations/versions/568f77aefcb4_add_timecode_to_authenticationlog.py

```diff
@@ -0,0 +1,85 @@
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '568f77aefcb4'
+down_revision = 'ff59dfd3bdf6'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('invite_code',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('code', sa.String(length=255), nullable=False),
+    sa.Column('expiration_date', sa.DateTime(), nullable=False),
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('code')
+    )
+    op.create_table('users',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('primary_username', sa.String(length=80), nullable=False),
+    sa.Column('display_name', sa.String(length=80), nullable=True),
+    sa.Column('password_hash', sa.String(length=512), nullable=True),
+    sa.Column('totp_secret', sa.String(length=255), nullable=True),
+    sa.Column('email', sa.String(length=255), nullable=True),
+    sa.Column('smtp_server', sa.String(length=255), nullable=True),
+    sa.Column('smtp_port', sa.Integer(), nullable=True),
+    sa.Column('smtp_username', sa.String(length=255), nullable=True),
+    sa.Column('smtp_password', sa.String(length=255), nullable=True),
+    sa.Column('pgp_key', sa.Text(), nullable=True),
+    sa.Column('is_verified', sa.Boolean(), nullable=True),
+    sa.Column('is_admin', sa.Boolean(), nullable=True),
+    sa.Column('show_in_directory', sa.Boolean(), nullable=True),
+    sa.Column('bio', sa.Text(), nullable=True),
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('primary_username')
+    )
+    op.create_table('authentication_logs',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('user_id', sa.Integer(), nullable=False),
+    sa.Column('successful', sa.Boolean(), nullable=False),
+    sa.Column('timestamp', sa.DateTime(), nullable=False),
+    sa.Column('otp_code', sa.String(length=6), nullable=True),
+    sa.Column('timecode', sa.Integer(), nullable=True),
+    sa.ForeignKeyConstraint(['user_id'], ['users.id'], ),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('secondary_usernames',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('username', sa.String(length=80), nullable=False),
+    sa.Column('user_id', sa.Integer(), nullable=False),
+    sa.Column('display_name', sa.String(length=80), nullable=True),
+    sa.ForeignKeyConstraint(['user_id'], ['users.id'], ),
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('username')
+    )
+    op.create_table('message',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('content', sa.Text(), nullable=False),
+    sa.Column('user_id', sa.Integer(), nullable=False),
+    sa.Column('secondary_user_id', sa.Integer(), nullable=True),
+    sa.ForeignKeyConstraint(['secondary_user_id'], ['secondary_usernames.id'], ),
+    sa.ForeignKeyConstraint(['user_id'], ['users.id'], ),
+    sa.PrimaryKeyConstraint('id')
+    )
```

Comment:
```suggestion
    
    op.add_column('authentication_logs', sa.Column('timecode', sa.Integer(), nullable=True))
```
You don't need to recreate the entire schema with this migration, you just need to add the column to the existing table